utils.py

```diff
@@ -147,8 +147,22 @@ def merge_mutation_sets(
   return to_return
 
 
-def get_mutation_positions(sequence,
-                           parent):
+def merge_multiple_mutation_sets(mutation_sets: Sequence[Tuple[Tuple[int, int], ...]]) -> List[Tuple[Tuple[int, int], ...]]:
```

Comment:
I expect that you can replace the entire body of this function with something like:
functools.reduce(merge_mutations_sets, mutation_sets, [])

Reply:
Yeah I'll think about this a bit more. It's not quite a reduce, because the result of merge_mutation_sets(A, B) is a list [AB1, AB2, ...], and then merge_mutation_sets([AB1, AB2,...], C) is not well defined

Reply:
I think this is accomplished now